examples/twri/scripts/GaussLens_inversion.jl

```diff
@@ -0,0 +1,92 @@
+################################################################################
+#
+# Run inversion for the Gaussian lens model
+#
+################################################################################
+
+R = Float32
+
+### Module loading
+using LinearAlgebra, JLD2, PyPlot, Dates, Distributed
+using Optim, LineSearches
+using JUDI, JUDI.TimeModeling, JUDI.SLIM_optim
+
+### Load synthetic data
+base_path = dirname(pathof(JUDI))*"/../examples/twri/"
+@load string(base_path*"data/GaussLens_data_acou.jld") model_true dat fsrc
```

Comment:
A small note somewhere is needed that describes in which order the scripts should be run. Perhaps somewhere in `examples/twri`

Reply:
Good point, replaced by a `try/catch` to generate data if doesn't exist.